ArjunaJTS/orbportability/classes/com/arjuna/orbportability/internal/utils/InitLoader.java

```diff
@@ -77,11 +78,19 @@ private void createInstance (String attrName, String className)
         {
             opLogger.i18NLogger.warn_internal_utils_InitLoader_exception(initName, e2);
         }
+	catch (InvocationTargetException e)
+	{
+	    throw new RuntimeException( e );
+	}
+	catch (NoSuchMethodException e)
+	{
+	    throw new RuntimeException( e );
+	}
```

Comment:
I suppose here we want to report the exception we caught using `opLogger.i18NLogger.warn_internal_utils_InitLoader_exception`. Actually, we can even group all exceptions in one catch:
```suggestion
	catch (IllegalAccessException | InstantiationException | InvocationTargetException | NoSuchMethodException e)
        {
            opLogger.i18NLogger.warn_internal_utils_InitLoader_exception(initName, e);
        }
```